classes/tombstone/DataObjectTombstoneSettingsDAO.inc.php

```diff
@@ -34,7 +34,7 @@ function &getSetting($tombstoneId, $name, $locale = null) {
 		while (!$result->EOF) {
 			$row = $result->getRowAssoc(false);
 			$value = $this->convertFromDB($row['setting_value'], $row['setting_type']);
-			if ($row['locale'] == '') $setting[$name] = $value;
+			if (!array_key_exists('locale', $row) || $row['locale'] == '') $setting[$name] = $value;
```

Comment:
I don't know what's possible to receive from `$result->getRowAssoc(false)`, but maybe to be safer to use `empty($row['locale'])`. Whether an empty string or null or false, an empty value shouldn't be used as an array key, right?